bin/wasabi.sh

```diff
@@ -272,10 +274,11 @@ package() {
     done; \
     sed -i '' -e "s|http://localhost:8080|${server}|g" target/constants.json 2>/dev/null; \
     sed -i '' -e "s|VERSIONLOC|${version}|g" target/app/index.html 2>/dev/null; \
-    if [[ "${WASABI_OS}" == "${WASABI_OSX}" || "${WASABI_OS}" == "${WASABI_LINUX}" ]]; then \
+# SCOTT and SEBASTIAN
```

Comment:
good call.

@longdogz 
note: there is a commented out trailing '... grunt test' ... do we need to include that?